configuration/apitokens.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: documentation
+title: openHAB API tokens
+---
+
+{% include base.html %}
```

Comment:
You can safely remove this line.
I have to remove all others too.
It is not needed for our current website builds.

Reply:
Ok! Indeed copied it from another page.